tensorboard/webapp/widgets/card_fob/card_fob_controller_component.ts

```diff
@@ -95,7 +95,9 @@ export class CardFobControllerComponent {
   }
 
   startDrag(fob: Fob, affordance: TimeSelectionAffordance, event: MouseEvent) {
-    this.stopEventPropagation(event);
+    if (affordance !== TimeSelectionAffordance.FOB) {
```

Comment:
What are your thoughts on using affordance for logic like this?

Reply:
One thing I cannot figure out from this logic is, why selecting other content like text and graph will not happen if I click on the fob? Logically this condition is not satisfied and stopEventPropagation is not trigged.

Reply:
After some testing and playing around I determined the contenteditable attribute stops the strange behavior from the event propagation. I added a comment trying to explain.